src/cli/verbosity.rs

```diff
@@ -12,18 +15,29 @@ pub struct Verbosity {
     verbose: usize,
 }
 
+fn from_verbosity(quiet: bool, verbose: usize) -> LevelFilter {
```

Comment:
Move to Verbosity as a private method.

I'd suggest something, but GitHub doesn't let me.

Reply:
oh, I don't know why I didn't do it in the first place. Thanks for catching it!